requirements/quality.txt

```diff
@@ -0,0 +1,8 @@
+# Requirements for code quality checks
+
+isort==4.3.4
```

Comment:
Can you pin `astroid==1.5.2`. For some reason when I run `make requirements` locally, it tries to install a version which requires Python 3.